platform/ios/src/MGLSDKUpdateChecker.mm

```diff
@@ -0,0 +1,37 @@
+#import "MGLSDKUpdateChecker.h"
+#import "NSBundle+MGLAdditions.h"
+
+@implementation MGLSDKUpdateChecker
+
++ (void)checkForUpdates {
+#if TARGET_IPHONE_SIMULATOR
+    // Abort if running in a playground.
+    if ([[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."]) {
```

Comment:
I later realized we bail in IB at the beginning of `-commonInit`. But this is a good check to add now that we're running code in the initialize method.